libmscore/chord.cpp

```diff
@@ -1168,7 +1168,14 @@ void Chord::processSiblings(std::function<void(Element*)> func) const
             func(ll);
 
       for (Note* note : _notes)
+      {
```

Comment:
Can you use the same indentation style than MuseScore?
See https://musescore.org/en/handbook/musescore-coding-rules

Reply:
Ooo, thanks. That thing : *Use six (6) spaces for indentation, no tabs.* frustrating me before. And i found solution (xml style scheme) in this doc =)
For sure i start use that.

Reply:
and banner style indenting

Reply:
This is still an issue.

Reply:
And still not banner style indenting...

Reply:
Fixed here now, but still wrong in bend.cpp, in multiple places

Reply:
Yes, i fix it tomorow, cause anyway need to write logic for separate chords duration for divisions with modulo (it make crash sometimes) and add logic for bends with zero pitch of two (or more)  points in tied bend.
This style.xml realy helps me with that =)